projects/dotcms-field-elements/src/components/dot-textfield/dot-texfield.e2e.ts

```diff
@@ -1,140 +1,345 @@
+                await page.waitForChanges();
+                const label = await dotTestUtil.getDotLabel(page);
+                expect(label.getAttribute('name')).toBe('text01');
+            });
+
+            it('should set name prop in dot-label with invalid value', async () => {
+                element.setProperty('name', { test: 'hi' });
+                await page.waitForChanges();
+                const label = await dotTestUtil.getDotLabel(page);
+                expect(label.getAttribute('name')).toBe('[object Object]');
+            });
+
+            it('should render when is a unexpected value', async () => {
+                element.setProperty('name', { input: 'text01' });
+                await page.waitForChanges();
+                expect(input.getAttribute('id')).toBe('dot-object-object');
+            });
+        });
+
+        describe('label', () => {
+            it('should set label prop in dot-label', async () => {
+                element.setProperty('label', 'Name:');
+                await page.waitForChanges();
+                const label = await dotTestUtil.getDotLabel(page);
+                expect(label.getAttribute('label')).toBe('Name:');
+            });
+            it('should set label prop in dot-label when is a unexpected value', async () => {
+                element.setProperty('label', [1, 2, 'test']);
+                await page.waitForChanges();
+                const label = await dotTestUtil.getDotLabel(page);
+                expect(label.getAttribute('label')).toEqual('1,2,test');
+            });
+        });
+
+        describe('placeholder', () => {
+            it('should set placeholder correctly', async () => {
+                element.setProperty('placeholder', 'Test');
+                await page.waitForChanges();
+                expect(input.getAttribute('placeholder')).toBe('Test');
+            });
+            it('should set placeholder correctly with invalid value', async () => {
+                element.setProperty('placeholder', { test: 'hi' });
+                await page.waitForChanges();
+                expect(input.getAttribute('placeholder')).toBe('[object Object]');
+            });
+        });
+
+        describe('hint', () => {
+            it('should set hint correctly', async () => {
+                element.setProperty('hint', 'Test');
+                await page.waitForChanges();
+                expect((await dotTestUtil.getHint(page)).innerText).toBe('Test');
+            });
+
+            it('should not render hint', async () => {
+                expect(await dotTestUtil.getHint(page)).toBeNull();
+            });
+
+            it('should not break hint with invalid value', async () => {
+                element.setProperty('hint', { test: 'hint' });
+                await page.waitForChanges();
+                expect((await dotTestUtil.getHint(page)).innerText).toBe('[object Object]');
+            });
+        });
+
+        describe('required', () => {
```

Comment:
Here, you're not checking that you passing the required to the `dot-label` as an attr.